app/src/main/res/values/strings.xml

```diff
@@ -54,7 +54,7 @@
     <!-- activity upload -->
     <string name="activity_upload_upload_activity_title">New Activity</string>
     <string name="activity_upload_edit_text_title_hint">Title</string>
-    <string name="activity_upload_edit_text_nparticipants_hint">Number of participants</string>
```

Comment:
I just see in edit profile, it's write two time Phone, instead of Description

Reply:
That's strange, I have one phone and one description on my side

Reply:
@RoxaneBurri yes, I modified this in my branch of this week (246_firebase-storage-abstraction). In fact there was an error in `strings.xml` and I modified this in order to have the proper "header texts" in EditProfileActivity 👍